gulpfile.js

```diff
@@ -0,0 +1,57 @@
+var gulp = require("gulp");
+
+var mochaPhantomJS = require('gulp-mocha-phantomjs');
+gulp.task("test", ["build-debug"], function() {
+  return gulp.src("tests/SpecRunner.html")
+    .pipe(mochaPhantomJS());
+});
+
+var webserver = require("gulp-webserver");
+gulp.task("test-browser", ["build-debug"], function() {
+  gulp.src(".")
+    .pipe(webserver({
+      livereload: true,
+      open: "tests/SpecRunner.html"
+    }));
+});
+
+var browserify = require("browserify");
+var source = require("vinyl-source-stream");
+gulp.task("build-release", ["jshint"], function() {
+  var bundler = new browserify({
+    debug: true, // Add source maps to output to allow minifyify convert them to minified source maps
+    standalone: "amd-resolver",
+    detectGlobals: false
```

Comment:
Ideally, these options would be in a variable/constant declared outside of both `build-release` and `build-debug` tasks. I prefer to wait on that until we decide to either modularize the tasks into their own files or leave them as one file. This waiting is mostly for formatting reasons.